src/plugins/opensearch_dashboards_utils/public/index.ts

```diff
@@ -71,6 +71,8 @@ export {
   getStatesFromOsdUrl,
   setStateToOsdUrl,
   withNotifyOnErrors,
+  parseUrl,
```

Comment:
can we verify that these have tests behind them. just in case. if we expose them and people want to use them but somehow unknowingly change them then it could be a bad time.

Reply:
Is this sufficient? https://github.com/opensearch-project/OpenSearch-Dashboards/blob/79744fbbbc91fe1801c81c05a9d0854f271d957b/src/plugins/opensearch_dashboards_utils/public/state_management/url/parse.test.ts#L33